.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -6,47 +6,71 @@ labels: ''
 
-**Describe the bug**
-<!-- A clear and concise description of what the bug is. -->
+Before raising a bug/issue please check that it has not already been reported 
+by searching the existing https://github.com/darktable-org/darktable/issues on GitHub.
+
+Please be aware that most developers work on this project in their free time.
+The more information you can provide in the initial report the quicker we can help.
+Therefore please help to be as accurate and concise as possible!
+
+For further information, please see the https://github.com/darktable-org/darktable/wiki#reporting-bugs.
+---------------------------------------------------------------
+
+
+**Describe the bug/issue**
 
 
 **To Reproduce**
-<!-- Provide detailed steps that can reproduce the behavior, such as:
+_Please amend or remove, what is/isn't applicable_
 1. Go to '...'
 2. Click on '...'
 3. Scroll down to '...'
 4. See error
--->
-
 
 **Expected behavior**
-<!-- A clear and concise description of what you expected to happen. -->
 
 
 **Screenshots**
-<!-- If applicable, add screenshots to help explain your problem. -->
+_Please remove, if not applicable (e.g. no screenshots)_
+
+**Screencast**
+_Please remove, if not applicable (e.g. no screencast)_
+
+**Which commit intruduced the error**
+_If possible, please try using `git bisect`. Please remove, if you do not know, what bisect is, or if you do, but not applicable_
+
+_A bisect is much appreciated and can significantly simplify the developer's job._
+_HowTo: https://github.com/darktable-org/darktable/wiki#finding-bug-causes and https://www.youtube.com/watch?v=D7JJnLFOn4A_
+
+
 
 
-**Platform (please complete the following information):**
- - Darktable Version: <!-- [e.g. 2.6.0] -->
- - OS: <!-- [e.g. Windows 8.1, Gentoo Linux] -->
- - <!-- OpenCL activated or no? -->
- - <!-- Which graphics card and driver version -->
+**Platform**
+_Please fill as much as possible including "unknown" and remove those lines, which are clearly not applicable_
 
+* darktable version : e.g. 3.5.0+250~gee17c5dcc
+* OS                : e.g. Linux - kernel 5.10.2 / Win10 (Patchlevel) / OSx
+* Linux - Distro    : e.g. Ubuntu 18.4
+* Memory            : 
+* Graphics card     : 
+* Graphics driver   : 
+* OpenCL installed  : 
+* OpenCL activated  : 
+* Xorg              : 
+* Desktop           : 
+* GTK+              : 
+* gcc               : 
+* cflags            : 
+* CMAKE_BUILD_TYPE  : 
 
 **Additional context**
-<!-- Add any other context about the problem here, for example:
- - Can you reproduce with another Darktable version?
- - Can you reproduce with a RAW or Jpeg or both?
- - Are the steps above reproduce with a fresh edit (removing history)?
- - Attach an XMP if this is necessary
- - Did you compile Darktable yourself? If so which compiler was used, with what options?
- - Is the issue still present using an empty/new config-dir
--->
+_Please fill as much as possible and remove those lines, which are clearly not applicable__
```

Comment:
```suggestion
_Please provide any additional information you think may be useful, for example:_
```

Reply:
_Please fill as much as possible and remove those lines, which are clearly not applicable__

the "," between lines and which is absolutely not necessary,
"Please fill as much as possible and remove those lines which are clearly not applicable" would be the correct usage.